src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -112,6 +112,7 @@
     "jack-client": ("jack",),
     "kafka-python": ("kafka",),
     "lark-parser": ("lark",),
+    "launchdarkly-server-sdk": ("ldclient"),
```

Comment:
I'm not 100% I got the order right here - the package on PyPI is called `launchdarkly-server-sdk`, but the top level directory in the package is called `ldclient`. Is this the right way around?

Reply:
Yeah, this is the right order. 👍 

One minor change: these have to be tuples, and creating a single-element tuple requires the trailing comma, unlike every other size of tuple (subtle!):

```suggestion
    "launchdarkly-server-sdk": ("ldclient",),
```

Reply:
Thanks, good catch!